### DISCUSSION TITLE
Support for Linux Ubuntu Server with no GUI

### DISCUSSION THREAD

**Discussion by csigristgit** [rustdesk/rustdesk#10341] [Q&A]
I have a lot of Windows machines connected with my self-hosted rustdesk Server. Now i like to add a few Ubuntu Servers. These has no GUI (Only CLI) and no Monitor attached (it's a VM). I really want to integrate this server in the rustdesk environment, so i had no other toolt to use. I simply wanna connect two Ports like SSH and Web. 

This should be possible, no?

**Comment by Poordeveloper** [comment]
https://github.com/rustdesk/rustdesk/wiki/Headless-Linux-Support

**Reply by csigristgit** [reply]
Allready seen this. This forces me to install ubuntu Desktop, which is a huge package. I Don't wanna install such a package on a server-only machine.

Any other sollutions?

**Comment by xlionjuan** [comment]
Tailscale or Cloudflare Tunnel if you need to remote control a pure CLI machine.
